findme-location-riddles/handler.py

```diff
@@ -76,14 +67,27 @@ def get_location_riddles_by_location_riddle_id(location_riddle_id: Annotated[int
     return location_riddles_service.get_location_riddle(location_riddle_id)
 
 
+@app.patch("/location-riddles/<location_riddle_id>/rate")
```

Comment:
I would use POST /rate and pass {rating: float} as body, query string param score seems a bit misleading for me and for me I would argue we post a new rating with this endpoint (maybe put can be argued too)

Reply:
i found this 'PATCH is used to apply partial updates to a resource, meaning that only the fields that need to be changed are sent in the request body.' online and that is why i wanted to do a patch here, also considering that we do not post a new resource but only update the existing location-riddle. @flyingbeat what do you think?

Reply:
what would be the benefit of passing a body instead of a query parameter? Do you think i should change it or is it okay like this?

Reply:
Okay 👍🏼 lets keep patch but query parameters should typically be used to specify requested data and not pass data that is uploaded or saved to a DB, I think: https://stackoverflow.com/a/25405668